src/lightning_app/core/app.py

```diff
@@ -67,6 +68,9 @@ def __init__(
                 This can be helpful when reporting bugs on Lightning repo.
             info: Provide additional info about the app which will be used to update html title,
                 description and image meta tags and specify any additional tags as list of html strings.
+            root_path: Set this to `/path` if you want to run your app behind a proxy at `/path` leave empty for "/".
```

Comment:
can you clarify what's proxy in this context? Our users might not know what would that be? Perhaps an example?

Also, in the next line, it is mentioned that the app won't start but not clear what should be done for it to actually work!

Reply:
I agree that the second sentence should probably point to some resource for further details :)

Reply:
@pritamsoni-hsr

Reply:
can we add this url? https://www.fortinet.com/resources/cyberglossary/proxy-server